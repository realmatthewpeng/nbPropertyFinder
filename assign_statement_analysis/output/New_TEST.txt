Total number of lines found: 0


Incorrect Lines:


Precision = 0
Recall = 0.0

x[x==-999]=np.NaN
z=x[:,~np.isnan(x).any(axis=0)]
x_test[x_test==-999]=np.NaN
z_test=x_test[:,~np.isnan(x_test).any(axis=0)]
