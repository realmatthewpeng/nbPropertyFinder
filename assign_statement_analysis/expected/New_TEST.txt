x[x==-999]=np.NaN
z=x[:,~np.isnan(x).any(axis=0)]
x_test[x_test==-999]=np.NaN
z_test=x_test[:,~np.isnan(x_test).any(axis=0)]